Weather App/Common/Extensions/UIImageExtension.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  UIImageExtension.swift
+//  Weather App
+//
+//  Created by Ivan Stajcer on 06.09.2021..
+//
+
+import UIKit
+
+extension UIImage {
+    func scalePreservingAspectRatio(targetSize: CGSize) -> UIImage {
```

Comment:
Why is this extension needed?

Reply:
It is not, after you pointed this out, I found a way to do same thing by modifying the constraints of the image and setting content mode to `.scaleToAspectFill`